packages/drivers/routerlicious-driver/src/errorUtils.ts

```diff
@@ -65,8 +66,10 @@ export function createR11sNetworkError(
             // If a service is temporarily down or a browser resource limit is reached, RestWrapper will throw
             // a network error with no status code (e.g. err:ERR_CONN_REFUSED or err:ERR_FAILED) and
             // the error message will start with NetworkError as defined in restWrapper.ts
-            return new GenericNetworkError(
-                errorMessage, errorMessage.startsWith("NetworkError"), props);
+            if (errorMessage.includes("failed, reason: self signed certificate")) {
```

Comment:
Let's update the comment above this to mention the fact that there are specific network errors which are non-retryable and need to be handled accordingly, and everything else can be handled as we currently do.

Reply:
Comment added for the `case undefined` when the error message is triggered by the `self signed certificate` error.